### PR TITLE
Add internal help for `dmhj` ##cons

### PR DESCRIPTION
**Pull request by officialcjunior**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Internal help for `dmhj` was missing inside the help for `dmh?`. I've added that in this Pull Request.

**Test plan**

See if there's a change which needs to be addressed.

**Closing issues**

None

### CODE REVIEW

File: libr/core/linux_heap_glibc.c [hunk 1]
```diff
@@ -1448,7 +1448,7 @@ void GH(print_malloc_info)(RCore *core, GHT m_state, GHT malloc_state) {
 
 static const char* GH(help_msg)[] = {
 	"Usage:", " dmh", " # Memory map heap",
-	"dmh", "", "List chunks in heap segment",
+	"dmh", "", "List the chunks inside heap segment",
```
Comment (trufae) [inline libr/core/linux_heap_glibc.c:1451]: inside **the** heap

File: libr/core/linux_heap_glibc.c [hunk 2]
```diff
@@ -1460,6 +1460,7 @@ static const char* GH(help_msg)[] = {
 	"dmhg", "", "Display heap graph of heap segment",
 	"dmhg", " [malloc_state]", "Display heap graph of a particular arena",
 	"dmhi", " @[malloc_state]", "Display heap_info structure/structures for a given arena",
+	"dmhj", "", "List the chunks inside heap segment in JSON format",
```
Comment (trufae) [inline libr/core/linux_heap_glibc.c:1463]: inside **the** heap

**Review by trufae — CHANGES_REQUESTED** [review]

File: libr/core/linux_heap_glibc.c (hunk 2 above)
Comment (officialcjunior) [reply libr/core/linux_heap_glibc.c:1463]: Sure. Will update in a minute.

Comment (officialcjunior) [reply libr/core/linux_heap_glibc.c]: Fixed. Marking this as resolved.

**Review by trufae — APPROVED** [review]

**Comment by officialcjunior** [comment]
Hey @XVilka, this one is done. 👍🏻

**Review by XVilka — APPROVED** [review]

**Pull request merged**